src/Currency.js

```diff
@@ -0,0 +1,257 @@
+const tokens = {
```

Comment:
i guess we should really pull this into its own library e.g. `@makerdao/currency`, huh